From 182c9ea522b762952e6774eda691eb4ef8abe2ab Mon Sep 17 00:00:00 2001
From: Richard W.M. Jones <rjones@redhat.com>
Date: Thu, 20 Jan 2011 16:00:59 +0000
Subject: [PATCH 15/18] Add /sys/hypervisor files from RHEL 5 Xen DomU to EXTRA_DIST.

This updates commit 55cf13ae4a00c30b55fe508f544406dce05bdb8d.

Cherry picked from commit 75a67daafc9a2aa6ec614a0aabb3be9236910cae.
---
 Makefile.am |   14 ++++++++++++++
 1 files changed, 14 insertions(+), 0 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 601a679..5d6563b 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -63,6 +63,20 @@ EXTRA_DIST = \
 	tests/rhel5-xen-domU-pv/sbin/dmidecode \
 	tests/rhel5-xen-domU-pv/sbin/id \
 	tests/rhel5-xen-domU-pv/sbin/virt-what-cpuid-helper \
+	tests/rhel5-xen-domU-pv/sys/hypervisor/properties/pagesize \
+	tests/rhel5-xen-domU-pv/sys/hypervisor/properties/changeset \
+	tests/rhel5-xen-domU-pv/sys/hypervisor/properties/virtual_start \
+	tests/rhel5-xen-domU-pv/sys/hypervisor/properties/capabilities \
+	tests/rhel5-xen-domU-pv/sys/hypervisor/properties/writable_pt \
+	tests/rhel5-xen-domU-pv/sys/hypervisor/type \
+	tests/rhel5-xen-domU-pv/sys/hypervisor/vmcoreinfo \
+	tests/rhel5-xen-domU-pv/sys/hypervisor/version/minor \
+	tests/rhel5-xen-domU-pv/sys/hypervisor/version/major \
+	tests/rhel5-xen-domU-pv/sys/hypervisor/version/extra \
+	tests/rhel5-xen-domU-pv/sys/hypervisor/compilation/compiler \
+	tests/rhel5-xen-domU-pv/sys/hypervisor/compilation/compiled_by \
+	tests/rhel5-xen-domU-pv/sys/hypervisor/compilation/compile_date \
+	tests/rhel5-xen-domU-pv/sys/hypervisor/uuid \
 	tests/test-baremetal.sh \
 	tests/test-esx4.1.sh \
 	tests/test-kvm.sh \
-- 
1.7.1

